cli/tools/fmt.rs

```diff
@@ -213,9 +255,16 @@ fn files_str(len: usize) -> &'static str {
   }
 }
 
-fn get_config() -> dprint::configuration::Configuration {
-  use dprint::configuration::*;
-  ConfigurationBuilder::new().deno().build()
+fn get_typescript_config(
+) -> dprint_plugin_typescript::configuration::Configuration {
+  dprint_plugin_typescript::configuration::ConfigurationBuilder::new()
+    .deno()
+    .build()
+}
+
+fn get_markdown_config() -> dprint_plugin_markdown::configuration::Configuration
+{
+  dprint_plugin_markdown::configuration::ConfigurationBuilder::new().build()
```

Comment:
We might want to have a discussion on what configuration would be best. Right now, deno's repo does `"textWrap": "always"` (https://github.com/denoland/deno/blob/master/.dprintrc.json), but the default is "maintain".